Gemfile.lock

```diff
@@ -44,22 +44,22 @@ GEM
       i18n (>= 0.7, < 2)
```

Comment:
This isn't relevant to the issue. IMO we should keep Gemfile updates separate or when required.